views/pages/results.html

```diff
@@ -0,0 +1,13 @@
+<!-- Results Page -->
+{{ define "content" }}
+<div class="relative h-[300px]">
```

Comment:
ok try to get everything that AJ has worked on involving results to work so we don't have to redo that